ca/ca.go

```diff
@@ -659,8 +663,70 @@ func (ca *CertificateAuthorityImpl) generateOCSPAndStoreCertificate(
 		// changes here, you should make sure they are reflected in orphan-finder.
 		ca.log.AuditErrf("Failed RPC to store at SA, orphaning certificate: serial=[%s] cert=[%s] err=[%v], regID=[%d], orderID=[%d]",
 			core.SerialToString(serialBigInt), hex.EncodeToString(certDER), err, regID, orderID)
+		if ca.orphanQueue != nil {
+			ca.queueOrphan(&orphanedCert{
+				DER:      certDER,
+				OCSPResp: ocspResp,
+				RegID:    regID,
+			})
+		}
 		return core.Certificate{}, err
 	}
 
 	return core.Certificate{DER: certDER}, nil
 }
+
+type orphanedCert struct {
+	DER      []byte
+	OCSPResp []byte
+	RegID    int64
+}
+
+func (ca *CertificateAuthorityImpl) queueOrphan(o *orphanedCert) {
+	_, err := ca.orphanQueue.EnqueueObject(o)
+	if err != nil {
+		ca.log.Err(fmt.Sprintf("failed to queue orphan for integration: %s", err))
```

Comment:
For this and the other `Err` calls, you can use `Errf` to simplify by removing the `Sprintf`. And for this one, I think it should be an `AuditErrf`. This error will be a thing we want to alert on, similar to how we currently alert on the orphan certificate error. Probably also we should have a counter stat, which would allow us to move the alerting to Prometheus.